modules/Bio/EnsEMBL/Production/Pipeline/Common/AnalysisSetup.pm

```diff
@@ -225,23 +225,18 @@ sub production_updates {
       $self->param($property, $properties{$property});
      }      
    }
+   $sth = $dbh->prepare(
```

Comment:
We can't just remove the writability test; sometimes we do not want to update the production database (e.g. if testing, or if VectorBase run the pipeline). The way to do this is to provide a read-only connection, and this code will fail if that's all that is available.